composer.json

```diff
@@ -29,7 +29,8 @@
     "require-dev": {
         "psr/log": "~1.0",
         "phpcr/phpcr-api-tests": "2.1.4",
-        "symfony/console": "~2.0"
```

Comment:
should be in suggests however, saying that you then can use commands from jackalope/jackalope and this repo...